pandas/tests/scalar/period/test_period.py

```diff
@@ -106,7 +106,7 @@ def test_construction(self):
         assert i1 == i3
 
         i1 = Period("1982", freq="min")
-        i2 = Period("1982", freq="MIN")
+        i2 = Period("1982-01-01 00:00")
```

Comment:
why change this one completely? I'd suggest just to assert that the warning is raised and still check they're equal

Reply:
> why change this one completely? 
>
I wanted to do assert and I thought this is proper way to define i2. 

> I'd suggest just to assert that the warning is raised and still check they're equal
>
thanks, it's a good idea. I did as you suggested.